client/js/socket-events/init.js

```diff
@@ -42,12 +42,19 @@ socket.on("init", function(data) {
 
 		webpush.configurePushNotifications(data.pushSubscription, data.applicationServerKey);
 
+		slideoutMenu.enable();
+
+		const viewport = $("#viewport");
+
+		if (viewport.outerWidth() >= 768) {
+			slideoutMenu.toggle(storage.get("thelounge.state.sidebar") === "true");
```

Comment:
Do we need the `=== true`? Surely it'll work fine without?

Reply:
Note the string. false as a string still evaluates to true.

Reply:
Oh stupid.

The purest in me feels like saying use Json.parse then ...